lib/internal/Magento/Framework/Pricing/PriceCurrencyInterface.php

```diff
@@ -14,7 +14,11 @@
  */
 interface PriceCurrencyInterface
 {
-    const DEFAULT_PRECISION = 2;
+    /**
+     * @deprecated precision should be retrieved from current locale
+     * @see \Magento\Framework\Pricing\Price\PricePrecisionInterface::getPrecision
+     */
+    const DEFAULT_PRECISION = null;
```

Comment:
I'd keep the constant value unchanged, it's enough to deprecate the constant.

Reply:
@sivaschenko making this fix without changing this constant value would require many more code changes. This constant is used in many places in whole codebase (i.e. https://github.com/magento/magento2/blob/fbfac3e51730abe2b79c6072544bc9742fcf7606/app/code/Magento/Catalog/Model/ProductRender/FormattedPriceInfoBuilder.php#L64), because precision is third parameter of the `format` method.

And I changed this parameter to be `null` as default, so my idea was to:
- change const value to also `null` - so every place that uses it, just to pass it as default value to the methods I changed, will also work as expected
- deprecate const - so make a note across codebase that every it occurrence should be replaced with simply `null`, but maybe in some next PR, to not make it over complicated.

Reply:
I think eliminating this constant from the calls is a safer and more backward compatible change than changing the constant value, considering that we don't know how this constant might be currently used in extensions and customizations.